gecko_tests/new_config.rst

```diff
@@ -1,153 +0,0 @@
-Turning on Firefox tests for a new configuration
```

Comment:
https://firefox-source-docs.mozilla.org/testing/tests-for-new-config/index.html is currently just a pointer to this page, so we might need to update that first?

Reply:
Good catch! I'll port the docs over to firefox-source-docs first.